Dockerfile

```diff
@@ -8,9 +8,10 @@ ENV GODEBUG=netdns=go
 COPY --from=alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 
 LABEL org.label-schema.version=latest
-LABEL org.label-schema.vcs-url="https://github.com/josmo/drone-rancher.git"
+LABEL org.label-schema.vcs-url="https://github.com/croudsupport/drone-rancher.git"
 LABEL org.label-schema.name="Drone Rancher"
-LABEL org.label-schema.vendor="Josmo"
+LABEL org.label-schema.vendor="Croud"
 
```

Comment:
same ... probably doesn't need to be edited at all

Reply:
as above